### ISSUE TITLE
include_tasks are skipped

### ISSUE THREAD

**Issue by pieteravonts** [ansible-community/ansible-lint#362]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.6.3
  config file = None
  configured module search path = [u'/Users/PAvon1000248194/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/PAvon1000248194/.local/share/virtualenvs/20180822-kCpl1wg8/lib/python2.7/site-packages/ansible
  executable location = /Users/PAvon1000248194/.local/share/virtualenvs/20180822-kCpl1wg8/bin/ansible
  python version = 2.7.14 (default, Nov  2 2017, 16:04:18) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
ansible-lint 3.4.23
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

I want to have a rule for checking include_tasks tasks. However, they are removed from the task list at the moment.

# Actual Behaviour (Bug report only)

playbook.yml
```yaml
---
- hosts: localhost
  gather_facts: no
  tasks:
    - debug:
      msg: hello
    - include_tasks: tasks.yml
    - debug:
      msg: bye
```
tasks.yml
```yaml
---
- debug:
    msg: included task
```

output of `ansible-playbook playbook.yml`
```
PLAY [localhost] *****************************************

TASK [debug] *****************************************
ok: [localhost] => {
    "msg": "hello"
}

TASK [include_tasks] *****************************************
included: tasks.yml for localhost

TASK [debug] *****************************************
ok: [localhost] => {
    "msg": "included task"
}

TASK [debug] *****************************************
ok: [localhost] => {
    "msg": "bye"
}

PLAY RECAP *****************************************
localhost                  : ok=4    changed=0    unreachable=0    failed=0
```

rule.py (simple rule only to visualise the task list)
```python
from ansiblelint import AnsibleLintRule

class rule(AnsibleLintRule):
    id = 'rule'
    tags = []

    def matchtask(self, file, task):
        print task['action']['__ansible_module__']
```

output of `ansible-lint -r . playbook.yml`
```
debug
debug
debug
```

desired output of `ansible-lint -r . playbook.yml`
```
debug
include_tasks
debug
debug
```

**Comment by willthames** [comment]
Actually I want this too, because there are likely a few things that rules could catch related to the recent changes to import_tasks, include_tasks, etc.

**Comment by willthames** [comment]
I've labelled it needs_implementation in case anyone wishes to do this, but I'll try and take a look at it myself when I have some time

**Comment by MatrixCrawler** [comment]
I thought i fixed that with #285 
It seems that the fix got lost with commit c03c3e8

**Comment by MatrixCrawler** [comment]
Made a new PR for that
https://github.com/ansible/ansible-lint/pull/425

**Comment by webknjaz** [comment]
The report is about Ansible 2.6 but that PR only fixes retro versions of Ansible (<2) so it probably doesn't really fix this report.

**Comment by MatrixCrawler** [comment]
@webknjaz yep. just wanted to notify you about that. It seems that the import_tasks, include_tasks, import_playbook and inclue are filtered from the tasks list on purpose if i get this expression correctly

`    return [task for task in tasks if
            set(['include', 'include_tasks',
                'import_playbook', 'import_tasks']).isdisjoint(task.keys())]`

**Comment by webknjaz** [comment]
yes, you understand it correctly

**Comment by MarkusTeufelberger** [comment]
Just please also be aware that `include_tasks` also can have a `file:` attribute instead of just the included file
